ingestion/functions/common/common_lib.py

```diff
@@ -47,7 +47,9 @@ def create_upload_record(env, source_id, headers, cookies):
                         headers=headers)
     if res and res.status_code == 201:
         res_json = res.json()
-        # TODO: Look for "errors" in res_json and handle them in some way.
+        # Upload POST API doesn't provide 'errors' json in response body, all
+        # we have is the response body itself which is enough to describe the
+        # error.
```

Comment:
Sorry, having a little trouble grokking this. Meaning `res.text` tells us the error, or?

Reply:
yes, but re-reading this, those comments server no purpose really it's more confusing than without them. Removed.